scripts/build-desktop-release.ps1

```
@@ -0,0 +1,274 @@
+[CmdletBinding()]
+param(
+    [Parameter(Mandatory = $true)]
+    [ValidatePattern('^\d+\.\d+\.\d+$')]
+    [string]$Version,
+
+    [string]$ElectronPath = (Join-Path (Split-Path -Parent (Split-Path -Parent $PSScriptRoot)) 'N.E.K.O.-PC'),
+
+    [ValidateSet('auto', 'windows', 'macos', 'linux')]
+    [string]$Platform = 'auto',
```

PSScriptAnalyzer (1.25.0)

[warning] The parameter 'Platform' has been declared but not used.

(PSReviewUnusedParameter)